packages/generators/generators/lib/files/ts/plugin/admin/src/components/Initializer/index.tsx

```diff
@@ -12,8 +12,7 @@ type InitializerProps = {
 };
 
 const Initializer: React.FC<InitializerProps> = ({ setPlugin }) => {
-  const ref = useRef<(id: string) => void | null>(null);
-  ref.current = setPlugin;
+  const ref = useRef<(id: string) => void | null>(setPlugin);
```

Comment:
```suggestion
  const ref = useRef(setPlugin);
```

I think this doesn't need typing because it won't even be `null` now we're setting it here. The type should therefore be inferred correctly.

Reply:
Thats true! 😄